commit 1e2fe722c41d4cc34094afb157b3eb06b4a50972
Author: David Zafman <dzafman@redhat.com>
Date:   Fri Dec 20 15:53:49 2019 -0800

    osd_stat_t::dump: Add option for ceph-mgr pythonn callers to skip ping network
    
    Due to the overhead of this on large clusters, we removed the network
    information was wasn't getting used anyway.
    
    Fixes: https://tracker.ceph.com/issues/43364
    
    Signed-off-by: David Zafman <dzafman@redhat.com>

diff --git a/src/mgr/ActivePyModules.cc b/src/mgr/ActivePyModules.cc
index 5595a2e6607..e30da4cb0dd 100644
--- a/src/mgr/ActivePyModules.cc
+++ b/src/mgr/ActivePyModules.cc
@@ -373,7 +373,7 @@ PyObject *ActivePyModules::get_python(const std::string &what)
     cluster_state.with_pgmap(
         [&f, &tstate](const PGMap &pg_map) {
       PyEval_RestoreThread(tstate);
-      pg_map.dump_osd_stats(&f);
+      pg_map.dump_osd_stats(&f, false);
     });
     return f.get();
   } else if (what == "osd_pool_stats") {
